### DISCUSSION TITLE
Literally can't delete notes in the charting editor (1.5.4)

### DISCUSSION THREAD

**Discussion by ACrazyTown** [KadeArchive/Kade-Engine#2053] [Q&A]
I'm using KE 1.5.4 to make a mod, and I literally cannot delete notes in the charting editor. There's like a 20% chance that it'll actually work and delete it. Is there any workaround to this (that's not switching to a newer version? I've switched versions like 2 times due to multiple issues)?

**Comment by NeeEoo** [comment]
I believe [this](https://github.com/KadeDev/Kade-Engine/commit/ececb4c3b2a2bcd5e23729b87c5c242edf608f79) is the code you want to fix this bug.
